cmake/targets/BuildBun.cmake

```diff
@@ -567,7 +567,7 @@ register_command(
       -Doptimize=${ZIG_OPTIMIZE}
       -Dcpu=${ZIG_CPU}
       -Denable_logs=$<IF:$<BOOL:${ENABLE_LOGS}>,true,false>
-      -Denable_asan=$<IF:$<BOOL:${ENABLE_ASAN}>,true,false>
+      # -Denable_asan=$<IF:$<BOOL:${ENABLE_ASAN}>,true,false>
```

Comment:
This hsould only be disabled in development by default. We should have another ENALBE_ZIG_ASAN flag.